src/main/java/net/hypixel/nerdbot/command/ItemGenCommand.java

```diff
@@ -79,11 +80,17 @@ public void askForInfo(
 
         // writing the rarity if the rarity is not none
         if (itemRarity != Rarity.NONE) {
+            // checks if there is a type for the item
             if (type == null || type.equalsIgnoreCase("none")) {
                 type = "";
             }
+            // checking if there is custom line break happening
+            if (handleLineBreaks == null || !handleLineBreaks.equalsIgnoreCase("true")) {

```

Comment:
With the above change
```suggestion
            if (!handleLineBreaks) {
```